### PR TITLE
Split the PhotoField from the main VIC bundle

### PR DESCRIPTION
**Pull request by jbalboni**
The VIC bundle is pretty big (170kb gzipped) and it's all loaded up front. One way of addressing this is to split out the PhotoField component, since that has some large-ish dependencies. Doing this drops 20kb off the initial bundle and loads the PhotoField when you get to that page. 

This works pretty well, since the photo cropper is below the fold on the majority of screens, so a user likely won't even notice the brief loading spinner below the photo instructions.

Additionally, I added a [bundle analyzer tool](https://github.com/webpack-contrib/webpack-bundle-analyzer) to our scripts, since I try to run that regularly.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -21,6 +21,7 @@
       "presets": ["env", "stage-2"],
       "plugins": [ 
         "istanbul",
+        "dynamic-import-node",
```
Comment (jbalboni) [inline .babelrc]: This converts `import()` syntax to `require()` statements when running tests in Node.

File: src/js/vic-v2/config/form.js [hunk 2]
```diff
@@ -199,7 +199,8 @@ const formConfig = {
                 };
               }
             }), {
-              'ui:field': PhotoField,
+              'ui:field': asyncLoader(() => import(/* webpackChunkName: "photo-field" */'../components/PhotoField')
```
Comment (rianfowler) [inline src/js/vic-v2/config/form.js]: pretty cool

**Review by rianfowler — APPROVED** [review]

**Review by cehsu — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -40,6 +40,9 @@ very secret.
 | add new npm modules                      | `yarn add my-module --dev`. There are no non-dev modules here. |
 | get the latest json schema               | `yarn remove vets-json-schema; yarn add https://github.com/department-of-veterans-affairs/vets-json-schema.git#{latest commit hash}` |
 | check test coverage                      | `npm run test:coverage`                  |
+| run bundle analyzer on our prodution JS bundles | `npm run build-analyze`                  |
```
Comment (U-DON) [inline README.md]: Typo: production.

**Review by U-DON — COMMENTED** [review]
webpack-bundle-analyzer changes look good.

**Pull request merged**